views/upload.njk

```diff
@@ -16,9 +23,11 @@
     <h1>Upload file here!</h1>         
     <div class="centerupload">
      <form action="/upload" id="form1" method="POST">
-      <input type='file' id="imgInp" accept="file"/*>
+      <input type='file' id="imgInp" accept="image"/*>
```

Comment:
Also there is still no `name` attribute. Please look and review all my comments carefully.